src/OpenTelemetry.Abstractions/Trace/ISpan.cs

```diff
@@ -117,5 +118,11 @@ public interface ISpan
         /// End the span.
         /// </summary>
         void End();
+
+        /// <summary>
+        /// End the span.
+        /// </summary>
+        /// <param name="endTimestamp">End UTC timestamp.</param>
```

Comment:
Why does it say UTC? in comment? DateTime type is smart enough I think

Reply:
If we are going to use `DateTime` it's important to always deal with `Utc` : https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime#the-datetime-structure to avoid ambiguity.
So we probably should even throw when non-UTC is set.